Contentstack.Core/Configuration/Config.cs

```diff
@@ -107,6 +108,10 @@ internal string HostURL
             {
                 if (Region == ContentstackRegion.EU)
                     return "cdn.contentstack.com";
+                if (Region == ContentstackRegion.Azure_EU)
+                    return "eu-cdn.contentstack.com";
+                if (Region == ContentstackRegion.Azure_NA)
+                    return "na-cdn.contentstack.com";
```

Comment:
same as above logic it is contact with region code and at the last it is same a you mentioned

Reply:
Implementation is incorrect: please check the documentation below.

https://www.contentstack.com/docs/developers/apis/content-management-api/

Make changes in the host on the basis of the Region selected and write a tests for all the possible regions available

Reply:
<img width="990" alt="Screenshot 2023-07-11 at 3 47 42 PM" src="https://github.com/contentstack/contentstack-dotnet/assets/139141060/f62237fc-49c2-46c1-bc74-2de2936d9694">
Just see this logic it is based on region it will concat the URL

I can not find any test cases created for Older One like for US and EU